src/AddIns/Uno.UI.Lottie/LottieVisualSource.Skottie.cs

```diff
@@ -471,5 +471,17 @@ private Size CompositionSize
 				? new Size(size.Width, size.Height)
 				: default;
 	}
+
+#if USE_HARDWARE_ACCELERATION
+	/// <remarks>
+	/// Important: <see cref="SKSwapChainPanel" /> does not arrange correctly on Uno.
+	/// Using this class, we can override its Arrange behaviour. This class can be
+	/// removed once SkiaSharp package depend on Uno 5 or later
+	/// </remarks>
+	public partial class UnoSKSwapChainPanel : SKSwapChainPanel
```

Comment:
@ajpinedam After thinking about it some more, it feels like this may break other scenarios in very subtle ways, as we're assuming that a framework element cannot have its own children. While true in a pure WinUI environment, in reality, many devs on iOS/Android have been using native children to host content.

I think we should revert the original change partially in `FrameworkElement`, and invoke the removed method only if the first child is not a `UIElement`. This will restore the behavior that we need, and will fix https://github.com/unoplatform/uno/pull/14547 at the same time.

Reply:
Changes completed.-